index.html

```diff
@@ -273,13 +273,15 @@ <h1 class="title">Contact me</h1>
         <p class="desc">If you have an application you are interested in developing, a feature that you need built or a
             project that needs coding. I’d love to help with it!</p>
 
-        <form action="https://formspree.io/f/xknayrnl" method="POST">
-            <input required placeholder="Name" maxlength="30" name="name" type="text" value="Lorem Ipsum" />
-            <input required placeholder="Email" name="email" type="email" value="loremipsum@gmail.com" />
-            <textarea rows="6" placeholder="Write your message here..." maxlength="500" required></textarea>
+        <form id="contact-form" action="https://formspree.io/f/xknayrnl" method="POST">
+            <input class="contact-form-name" required placeholder="Name" maxlength="30" name="name" type="text" value="Lorem Ipsum" />
+            <input class="contact-form-email" required placeholder="Email" name="email" type="email" value="loremipsum@gmail.com" />
+            <textarea class="contact-form-message"  rows="6" placeholder="Write your message here..." maxlength="500" required></textarea>
             <div class="contact-button">
-                <button class="button-outline">Get in touch</button>
+                <button id="contact-submit-btn" class="button-outline">Get in touch</button>
             </div>
+
+            <div id="error-message"></div>
```

Comment:
- [ ] Please note that after filling out the form, and not submitting it, the entered values should be saved in the browser 😄 
![from](https://github.com/yemidada/desktop-version-portfolio/assets/99656687/58a6f000-8b40-4612-94ae-77cc5be12d6d)

Reply:
@yemidada , there is a type in the image. Date should be data.

Reply:
Hello @wayungi We don't understand completely about the date is not saved as required in your screenshot what is mean and We are confused about that.